capa/features/extractors/dotnetfile.py

```diff
@@ -69,6 +70,11 @@ def extract_file_strings(pe: dnfile.dnPE, **kwargs) -> Iterator[Tuple[String, in
     yield from capa.features.extractors.common.extract_file_strings(pe.__data__)
 
 
+def extract_mixed_mode_characteristic_features(pe: dnfile.dnPE, **kwargs) -> Iterator[Tuple[Characteristic, int]]:
+    if not bool(pe.net.Flags.CLR_ILONLY):
```

Comment:
see `capa.features.extractors.dotnetfile.DotnetFileFeatureExtractor.is_mixed_mode`?

Reply:
let's move to check to a helper function and call from both locations? otherwise we need an instance of capa.features.extractors.dotnetfile.DotnetFileFeatureExtractor to call is_mixed_mode

Reply:
sounds good!